File: Mathematical/Java/Average_in_a_stream.java
class Solution {
    float[] streamAvg(int[] arr, int n) {
        float result[]= new float[n];
        float d=1;
        for (int i=0; i<arr.length; i++) {
            float sum=0;
            for (int j=0; j<=i; j++) {
                sum += arr[j];
            }
        sum=sum/d;
        result[i]=sum;
        ++d;
        }
        return result;
    }
}

File: Solutions/Python/Average_in_a_stream.py
class Solution:
	def streamAvg(self, arr, n):
		# code here
		l=[]
        sum=0
        for i in range(0,len(arr)):
            for j in range(i,i+1):
                sum=sum+arr[j]
                l.append(sum/(j+1))
        return l 
